change/@fluentui-react-43bf83e2-d32b-45f7-a346-1c6ea8947614.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28891&commitId=7903d502a857ca191de2d112844e0a8d504936ee&env=prod&clientType=FLUENTUI)


  <details><summary><b>Image</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.Fit- cover, image taller.default.chromium.png|7603| Changed|
|Image.Fit- contain, image taller.default.chromium.png|1186| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>